java/feathub-udf/flink-udf/src/test/java/com/alibaba/feathub/flink/udf/aggregation/AggFuncWithLimitWithoutRetractTest.java

```diff
@@ -0,0 +1,52 @@
+/*
+ * Copyright 2022 The FeatHub Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.alibaba.feathub.flink.udf.aggregation;
+
+import org.apache.flink.table.api.DataTypes;
+
+import org.junit.jupiter.api.Test;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+/** Test for {@link AggFuncWithLimitWithoutRetract}. */
+public class AggFuncWithLimitWithoutRetractTest {
```

Comment:
Let's merge this class to `AggFuncWithLimitWithoutRetractTest`, similar to tests for other aggFuncWithoutRetract classes.